### DISCUSSION TITLE
Are there any MS or SP settings that might affect PnP search?

### DISCUSSION THREAD

**Discussion by gunitavilsone** [microsoft-search/pnp-modern-search#2331] [Q&A]
I have a column with multiple people 'owner'.
Managed property 'RefinableString01' with Crawled Property 'ows_q_USER_owner'.

In our own test environment PnP search works perfectly, but within two individual customer tenants search doesn't find the entry associated with the people from 'owner' column.

PnP (Modern Search v4.7.0 - June 2022) setup and SP (Online) site with registers and even people accounts were done by me and in the same way - so I can't figure out where the issue might be.

**Comment by wobba** [comment]
It would help to know where you did the cp/mp mapping (site/tenant) and if you re-indexed or created content after the mapping was done.

**Reply by gunitavilsone** [reply]
Hello, apologies in advance if I'm not answering your question properly - I am a novice in SharePoint world and english is my second language so I do struggle with terminology..
However this is the route i follow while setting up SharePoint site and PnP search and on our site everything works perfectly - happy to show - but we have done the same thing for two customers and it doesnt work for them and as far as I can see all configurations are the same:
SP site 
1.   Create Public Teams Site
2.   Create List Columns via Site Settings, add to new collection
3.   Create Library or List and add columns from existing site columns 
4.   Insert a few sample values and Reindex Site

PnP Search 
1.   Create Search page 
2.   Create Query Settings and Insert Query for all Lists/Libraries 
_{searchTerms?} Path:"https://tenant.sharepoint.com/sites/sitename/Lists/listname/*"_ 
3.   Copy the Query Source ID “_70223e1e-f7b2-4bb3-804f-2cd6d7565d21_" 
4.   Insert PnP search web parts 
5.   Set up Search Results Webpart 
·       (page1) paste the ID in Result Source Id 
·       (page3) Input query text - Dynamic value/Page environment/Query string/Query parameters/q
6.   Connect Results Webpart to Filters Webpart 
7.   Set up Search Box Webpart            
·       (page3) YES to Dynamic data source/Page environment/Query string/Query parameters/q
8.   Configure Managed properties and Reindex Site
9.   In Search Results Webpart configure Selected Properties/RefinableString 
10. In Filters Webpart Customise Filters

**Reply by wobba** [reply]
Site columns are not by default refinable. You have to remap the crawled property to for example RefinableString02 for it to work as a filter. See https://docs.microsoft.com/en-us/microsoft-365/community/how-do-site-columns-become-managed-properties-thus-available-for-search which is linked via our web part docs at https://microsoft-search.github.io/pnp-modern-search/scenarios/set-up-managed-properties/

**Reply by wobba** [reply]
But if you have mapped correct, waited to ensure items are re-indexed (no SLA on this and can take from 2 min to 48h), and still no data in `RefinableString01`, then this is an indexing issue. And you should be able to repro it using classic web parts or Search Query Tool. If so log a support request with Microsoft.

**Comment by gunitavilsone** [comment]
Hello Mikael, i have been in touch with Microsoft support person regarding
the indexing issue and they still trying to figure it out. The last time
they asked me if you could collaborate regarding this..

Their email address is: ***@***.***

And case number is #:32905261.

I am happy to provide any information necessary and pay for your help.

I have two test sites under different users.. pnp search filter for one
site work.. for other one doesn't.

All setup is done identically. Filter needs to find individual user
associated with document even when in entry there are multiple users
associated with that document..
At the moment RefinableString00 ows_involved returns combinations of users
(as they are listed in the column). RefinableString01 ows_q_USER_involved
shows individual user, in url string shows how many documents are
associated with it, but doesn't show any documents in web results box.
I am happy to share login details with the test sites if needed.
Thank you in advance

**Reply by wobba** [reply]
Can you share how you have configured the cp mapping. On the site or tenant wide. And then where you are searching from. And are you using queries like `RefinableString01:value` when searching? Is it when you pick the refinement filter it breaks? If the latter it could be related to https://github.com/microsoft-search/pnp-modern-search/issues/2358 / https://github.com/SharePoint/sp-dev-docs/issues/8443

You have to be explicit on exactly what is not working.

**Reply by gunitavilsone** [reply]
cp mapping is configured within Site Collection Administration and then Result Source Query settings are within Site Administration. Here are screenshots:
![image](https://user-images.githubusercontent.com/105509409/191868322-233879cc-2d39-467b-8a5b-106f123d75a2.png)
![image](https://user-images.githubusercontent.com/105509409/191868389-6fe35fd5-d57e-4dcc-8505-04d4e755b969.png)
![image](https://user-images.githubusercontent.com/105509409/191869224-263a269b-164e-4f81-955d-bc83eadd4731.png)

I have set up both types and at the moment property RefinableString01 with 'ows_involved' return combinations of people and search works for these, however i need to be able to find all documents within site for each person individually where he is listed as 'involved'. The property RefinableString02 with 'ows_q_users_involved' return single person and it shows number of documents associated with this person, but doesnt return any results in the search results webpart.

p.s. i have 4 different tenants with individual Microsoft licences, all setup is identical, but for one tenant search works perfectly and 3 others have the issue stated above.

Thank you

**Reply by wobba** [reply]
Seems like an API/service issue and you have to log this with Microsoft support or via https://github.com/SharePoint/sp-dev-docs
